dll/win32/shell32/CShellLink.cpp

```diff
@@ -2802,6 +2802,29 @@ BOOL CShellLink::OnInitDialog(HWND hwndDlg, HWND hwndFocus, LPARAM lParam)
     if (m_sDescription)
         SetDlgItemTextW(hwndDlg, IDC_SHORTCUT_COMMENT_EDIT, m_sDescription);
 
+    /* Hot key */
+    SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_KEY_HOTKEY, HKM_SETHOTKEY, m_Header.wHotKey, 0);
+
+    /* Run */
+    const WORD runstrings[] = { IDS_SHORTCUT_RUN_NORMAL, IDS_SHORTCUT_RUN_MIN, IDS_SHORTCUT_RUN_MAX };
+    const BYTE runshowcmd[] = { SW_SHOWNORMAL, SW_SHOWMINNOACTIVE, SW_SHOWMAXIMIZED };
+    HWND hRun = GetDlgItem(hwndDlg, IDC_SHORTCUT_RUN_COMBO);
```

Comment:
```suggestion
    HWND hRunCombo = GetDlgItem(hwndDlg, IDC_SHORTCUT_RUN_COMBO);
```

Reply:
Does this really make the code better? All the related code is less than 15 lines away and it is very clear that it is a combobox because of the CB_ messages.

Reply:
Another code reader is not you.

Reply:
I agree with Katayama suggestions, going to apply them. Arguing with verified reviewers is not a good idea in any way, this only makes review process slower and delays the merge. Keep in mind, while reviewing, we are interested to help you getting the changes in.